2.5/Dockerfile.rhel7

```diff
@@ -0,0 +1,66 @@
+FROM rhscl/s2i-base-rhel7:1
+
+# This image provides a Ruby environment you can use to run your Ruby
+# applications.
+
+EXPOSE 8080
+
+ENV RUBY_MAJOR_VERSION=2 \
+    RUBY_MINOR_VERSION=5 \
+    RUBY_NODEJS_SCL="rh-nodejs6"
+
+ENV RUBY_VERSION="${RUBY_MAJOR_VERSION}.${RUBY_MINOR_VERSION}" \
+    RUBY_SCL_NAME_VERSION="${RUBY_MAJOR_VERSION}${RUBY_MINOR_VERSION}"
+
+ENV RUBY_SCL="rh-ruby${RUBY_SCL_NAME_VERSION}" \
+    IMAGE_NAME="rhscl/ruby-${RUBY_SCL_NAME_VERSION}-rhel7"
+
+ENV SUMMARY="Platform for building and running Ruby $RUBY_VERSION applications" \
+    DESCRIPTION="Ruby $RUBY_VERSION available as docker container is a base platform for \
+building and running various Ruby $RUBY_VERSION applications and frameworks. \
+Ruby is the interpreted scripting language for quick and easy object-oriented programming. \
+It has many features to process text files and to do system management tasks (as in Perl). \
+It is simple, straight-forward, and extensible."
+
+LABEL summary="$SUMMARY" \
+      description="$DESCRIPTION" \
+      io.k8s.description="$DESCRIPTION" \
+      io.k8s.display-name="Ruby ${RUBY_VERSION}" \
+      io.openshift.expose-services="8080:http" \
+      io.openshift.tags="builder,ruby,ruby${RUBY_SCL_NAME_VERSION},${RUBY_SCL}" \
+      com.redhat.component="${RUBY_SCL}-docker" \
+      name="${IMAGE_NAME}" \
+      version="${RUBY_VERSION}" \
+      usage="s2i build https://github.com/sclorg/s2i-ruby-container.git \
+--context-dir=${RUBY_VERSION}/test/puma-test-app/ ${IMAGE_NAME} ruby-sample-app" \
+      maintainer="SoftwareCollections.org <sclorg@redhat.com>"
+
+ENV INSTALL_PKGS=" \
```

Comment:
I dont think this needs to be declared as an ENV of the image - it will only be used once, during the install. Please move the declaration to the RUN command that does the installation.

For reference: https://github.com/sclorg/s2i-ruby-container/blob/master/2.4/Dockerfile.rhel7#L31

Reply:
OK, I will modify like this.
But I prefer the long line is wrapped by escape "\". How do you think?

```
INSTALL_PKGS="rh-ruby24 rh-ruby24-ruby-devel rh-ruby24-rubygem-rake rh-ruby24-rubygem-bundler rh-nodejs6 rh-nodejs6-npm" && \
```